task/redhat-codeready-dependency-analysis/0.1/samples/pipeline-run.yaml

```diff
@@ -0,0 +1,29 @@
+# PipelineRun for starting pipeline
+# Make sure Workspace is attached and Secret is created,
+# also configure 'pre-task-setup' and 'post-task-setup' tasks before running the pipeline.
+# refer examples from samples
+
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: redhat-codeready-dependency-analysis-pipeline
+spec:
+  pipelineRef:
+    name: redhat-codeready-dependency-analysis-pipeline
+  workspaces:
+    - name: output
+      persistentvolumeclaim:
+        claimName: output
+
+# Change link to git_url and manifest-file-name according to target repository
+  params:
+    - name: file_url
+      value: {{MANIFEST_FILE_URL}} # Replace it with URL of manifest file to be scanned for demo.
+    - name: pkg-installation-directory-path
+      value: site-package
+    - name: manifest-file-path
+      value: requirements.txt
+    - name: output-file-path
+      value: redhat-codeready-dependency-analysis-report.json
+    - name: image
+      value: ghcr.io/fabric8-analytics/crda-python:3.6
```

Comment:
3.6 is not going to be supported anymore by the python foundation any reasons why to use this ?

Reply:
Python 3.6 is only given as a default ecosystem/version. This is totally up to users to chose ecosystem/version according to his/her requirements. 
We will be providing images for all available versions of ecosystems we support (like Python3.6/3.7/3.8..., Golang 1.12/1.13/1.14/1.15... and similar for Maven and Node), users can select any image from list given [here](https://github.com/fabric8-analytics/crda-images) and use it in pipeline (Currently we are supporting only Python for Tekton task, that is why only Python images are available on repo. Images for Maven/Node/Go will be uploaded very soon).